src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="groups">
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">
+            Surname:
+            <input
+              class="form-field"
+              type="text"
+              name="surname"
+              id="surname"
+              required
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="name">
+            Name:
+            <input
+              type="text"
+              name="name"
+              id="name"
+              required
+            >
+          </label>
+        </div>
+        <div class="form-field">
```

Comment:
add an empty line between _multiline_ siblings
```suggestion
        </div>
        
        <div class="form-field">
```